joml-geometry/src/main/java/org/terasology/joml/geom/AABBd.java

```diff
@@ -130,6 +130,26 @@ public AABBd(double minX, double minY, double minZ, double maxX, double maxY, do
         this.maxZ = maxZ;
     }
 
+    /**
+     *
+     * @param minX the x coordinate of the minimum corner
+     * @param minY the y coordinate of the minimum corner
+     * @param minZ the z coordinate of the minimum corner
+     * @param maxX the x coordinate of the maximum corner
+     * @param maxY the y coordinate of the maximum corner
+     * @param maxZ the z coordinate of the maximum corner
```

Comment:
Is max inclusive or exclusive? I think we should state this _somewhere_, and ideally often enough that this info is obvious at some point :sweat_smile: